src/queue/consumers/oneidentity/utils/validateVisitMessage.ts

```diff
@@ -4,7 +4,8 @@ export function validateVisitMessage(message: any): VisitMessage | never {
   if (
     message?.visitorId === undefined ||
     message?.startAt === undefined ||
-    message?.endAt === undefined
+    message?.endAt === undefined ||
+    message?.proposal === undefined
   ) {
```

Comment:
suggestion:
Maybe we can use type narrowing here and rewrite this  method as 
```
export function isVisitMessage(message: any): message is VisitMessage {
  return (
    message != null &&
    typeof message === 'object' &&
    'visitorId' in message &&
    'startAt' in message &&
    'endAt' in message &&
    'proposal' in message
  );
}
```

I think this would make the usability of the function better downstream.